modules/react/popup/lib/PopupCard.tsx

```diff
@@ -62,6 +62,35 @@ const StyledPopupCard = styled(Card)<
   };
 });
 
+function getSpace(value?: string | number) {
+  if (value && value in space) {
+    return space[value as keyof typeof space];
+  } else {
+    return value;
+  }
+}
+
+function getMaxHeight(margin: PopupCardProps['margin']) {
+  // set the default margin offset to space.xl
+  let marginOffset: string | number = space.xl;
+
+  if (margin) {
+    // parse the margin prop
+    if (typeof margin === 'string') {
+      const marginValues = margin.split(' ');
+      const marginTop = getSpace(marginValues[0]);
+      // If provided, use the specific margin-bottom in the shorthand, otherwise use the margin-top value
+      const marginBottom = getSpace(marginValues[2]) || marginTop;
+
+      marginOffset = `(${marginTop} + ${marginBottom})`;
+    } else {
+      // if margin is a number, double it to get the offset
+      marginOffset = `${margin * 2}px`;
```

Comment:
Getting yarn typecheck error because `margin` is not a number type. 
```
modules/react/popup/lib/PopupCard.tsx:88:7 - error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

88       marginOffset = `${margin * 2}px`;
                           ~~~~~~
Found 1 error.
```
Can we do `Numeric(margin)` here or update the type of margin to string | numeric? Any suggestion?

Reply:
Yeah, TS is getting confused. I think it's easier to update the type of margin at the top.